pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -243,6 +244,13 @@ func (c *Cluster) ReconcileActiveMgrServices(daemonNameToUpdate string) error {
 	} else {
 		currentDaemon := svc.Spec.Selector[controller.DaemonIDLabel]
 		if currentDaemon == daemonNameToUpdate {
+			if corev1.ServiceTypeClusterIP == svc.Spec.Type {
```

Comment:
Does this code block have any side effect? If it's just logging, seems like we don't need it?